pyproject.toml

```diff
@@ -71,6 +71,9 @@ indent-style = "space"
 skip-magic-trailing-comma = false
 line-ending = "auto"
 
+[tool.mypy]
+exclude = ["venv"]
```

Comment:
this branch appears to be out of date with main. Make sure to merge main in (namely, noticed that missing exclusion of examples from documentation release).